packages/rest/src/rest-server.ts

```diff
@@ -449,7 +452,7 @@ export class RestServer extends Context implements Server {
     // Apply deep clone to prevent getApiSpec() callers from
     // accidentally modifying our internal routing data
     spec.paths = cloneDeep(this.httpHandler.describeApiPaths());
-
+    spec.definitions = this.httpHandler.getApiDefinitions();
```

Comment:
Should we apply `cloneDeep` here too?

Reply:
I thought it'd make sense when I wrote the code there, but it seems like `cloneDeep` is currently typed for just `PathsObject`. Maybe I'll add in a similar function for `DefinitionsObject`.